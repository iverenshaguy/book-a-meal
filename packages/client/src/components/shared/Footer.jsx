import React from 'react';

/**
 * @exports
 * @function Footer
 * @returns {JSX} Footer
 */
const Footer = () => (
  <div className="footer">
    <p>&copy; 2018. Iveren Shaguy. All images were gotten from the internet and are not mine.</p>
  </div>
);

export default Footer;
