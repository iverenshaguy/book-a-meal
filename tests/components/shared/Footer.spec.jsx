import React from 'react';
import Footer from '../../../src/components/shared/Footer';

describe('Footer', () => {
  it('should render the Footer component correctly', () => {
    const shallowWrapper = shallow(<Footer />);

    expect(toJson(shallowWrapper)).toMatchSnapshot();
  });
});
