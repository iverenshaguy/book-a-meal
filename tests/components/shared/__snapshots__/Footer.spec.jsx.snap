// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render the Footer component correctly 1`] = `
<div
  className="footer"
>
  <p>
    Â© 2018. Iveren Shaguy. All images were gotten from the internet and are not mine.
  </p>
</div>
`;
